### PR TITLE
[docs] fix 404 url

### PR DESCRIPTION
**Pull request by rinechran**
**Describe the pull request**

- #### What does your PR fix?
  Fixes docs packaging-zipfiles.md

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  all

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  `Your answer`

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  Yes / I am still working on this PR

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

**Review by JackBoosY — APPROVED** [review]

File: docs/examples/packaging-zipfiles.md [hunk 1]
```diff
@@ -11,12 +11,12 @@ Second, determine a suitable package name. This should be ASCII, lowercase, and
...
 Finally, if the server's name for the archive is not very descriptive (such as downloading a zipped commit or branch from GitHub), choose a nice archive name of the form `<packagename>-<version>.zip`.
 
-*`zlib1211.zip` is a fine name, so no change needed.*
+*`zlib1212.zip` is a fine name, so no change needed.*
 
 All this information can then be passed into the `create` command, which will download the sources and bootstrap the packaging process inside `ports/<packagename>`.
 
 ```no-highlight
-PS D:\src\vcpkg> .\vcpkg create zlib2 http://zlib.net/zlib-1.2.11.tar.gz zlib1211.tar.gz
+PS D:\src\vcpkg> .\vcpkg create zlib2 http://zlib.net/zlib-1.2.12.tar.gz zlib1212.tar.gz
```
Comment (JackBoosY) [inline docs/examples/packaging-zipfiles.md:19]: The following related documents should also be updated.

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: docs/examples/packaging-zipfiles.md (hunk 1 above)
Comment (JackBoosY) [reply docs/examples/packaging-zipfiles.md:19]: Did you forget to push your changes?

Comment (rinechran) [reply docs/examples/packaging-zipfiles.md:19]: I'm sorry. I forgot to push. I'll do it when I get home

**Comment by JackBoosY** [comment]
Close and reopen this PR to trigger CLA check.

**Review by JackBoosY — APPROVED** [review]

**Pull request merged**